source/transactions/transactions.rst

```diff
@@ -1298,12 +1298,18 @@ The following commands are allowed inside transactions:
 
 8.  aggregate (including $lookup)
 
-    -  The ``$out`` and ``$merge`` stages are prohibited.
+    -  The ``$out`` stage is only allowed if the target does not exist. The
```

Comment:
Did you consider a test for this?  It seems wise since some drivers have different code paths for aggregate operations that write via $out/$merge

Reply:
This was originally part of the scope but has since been removed. I removed the tests but forgot this change, so I removed it now.